umm-spec-lib/src/cmr/umm_spec/versioning.clj

```diff
@@ -10,7 +10,7 @@
   "A map of concept type to a sequence of valid UMM Schema versions, with the newest one last.
   The sequence must be updated when new schema versions are added for the concept type."
   {:collection ["1.0" "1.1" "1.2" "1.3" "1.4" "1.5" "1.6" "1.7" "1.8" "1.9" "1.10" "1.11" "1.12"
-                "1.13" "1.14" "1.15"]
+                "1.13" "1.14" "1.15" "1.15.1"]
```

Comment:
We might want a more elegant solution, I don't now how many minor version updates we are going to do but this vector might get out of hand.

Reply:
It doesn't need to be in this PR though

Reply:
I created issue https://bugs.earthdata.nasa.gov/browse/CMR-6249